cfy_manager/components/prometheus/prometheus.py

```diff
@@ -0,0 +1,319 @@
+#     logger.notice('Unpacking archive {0}'.format(archive_file_name))
+#     common.untar(archive_file_name, dest_dir)
+
+
+# def _deploy_exporter(exporter, src_dir):
+#     logger.notice('Copying {0} binaries'.format(exporter['description']))
+#     dest_file_name = join(BIN_DIR, exporter['name'])
+#     common.copy(join(src_dir, exporter['name']), dest_file_name)
+#     common.chown(CLOUDIFY_USER, CLOUDIFY_GROUP, dest_file_name)
+#     if 'deploy_config' not in exporter:
+#         return
+#     for file_name, dest_file_name in exporter['deploy_config'].items():
+#         files.deploy(join(src_dir, file_name), dest_file_name)
+#     common.chown(CLOUDIFY_USER, CLOUDIFY_GROUP, dest_file_name)
+
+
+# def _install_prometheus():
+#     logger.notice('Installing Prometheus...')
+#     _create_prometheus_directories()
+#     working_dir = _create_directory(join(sep, 'tmp', 'prometheus'))
+#     archive_file_name = _download_release(working_dir,
+#                                           'https://github.com/prometheus',
+#                                           PROMETHEUS,
+#                                           PROMETHEUS_VERSION)
+#     _unpack_archive(archive_file_name, working_dir)
+#     _copy_prometheus(working_dir)
+#     common.remove(working_dir)
+#     logger.notice('Prometheus successfully installed')
+
+
+def _create_prometheus_directories():
+    logger.notice('Creating Prometheus directories')
+    common.mkdir(PROMETHEUS_DATA_DIR)
+    common.chown(CLOUDIFY_USER, CLOUDIFY_GROUP, PROMETHEUS_DATA_DIR)
+    for dir_name in ('rules', 'exporters',):
+        dest_dir_name = join(PROMETHEUS_CONFIG_DIR, dir_name)
+        common.mkdir(dest_dir_name)
+        common.chown(CLOUDIFY_USER, CLOUDIFY_GROUP, dest_dir_name)
+
+
+# def _copy_prometheus(src_dir):
+#     logger.notice('Copying Prometheus binaries and default configuration')
+#     common.copy(join(src_dir, 'prometheus'), BIN_DIR)
+#     common.chown(CLOUDIFY_USER, CLOUDIFY_GROUP,
+#                  join(BIN_DIR, 'prometheus'))
+#     common.copy(join(src_dir, 'promtool'), BIN_DIR)
+#     common.chown(CLOUDIFY_USER, CLOUDIFY_GROUP,
+#                  join(BIN_DIR, 'promtool'))
+#     common.copy(join(src_dir, 'consoles'),
+#                 PROMETHEUS_CONFIG_DIR)
+#     common.copy(join(src_dir, 'console_libraries'),
+#                 PROMETHEUS_CONFIG_DIR)
+#     common.chown(CLOUDIFY_USER, CLOUDIFY_GROUP, PROMETHEUS_CONFIG_DIR)
+
+
+# def _install_exporters():
+#     for exporter in EXPORTERS:
+#         logger.notice('Installing {0}...'.format(exporter['description']))
+#         working_dir = _create_directory(join(sep, 'tmp', exporter['name']))
+#         archive_file_name = _download_exporter(exporter, working_dir)
+#         _unpack_archive(archive_file_name, working_dir)
+#         _deploy_exporter(exporter, working_dir)
+#         logger.notice(
+#             '{0} successfully installed'.format(exporter['description']))
```

Comment:
drop the commented code, and also plz note that `install` should be done by just installing rpms (see comment about sources.py) and not on the python side